apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -453,13 +453,12 @@ export default Vue.extend({
         {
           label: '<x-menuitem><x-label>Copy row(s) as SQL</x-label></x-menuitem>',
           action: async (_e, cell) => {
-
             const fixed = this.getCleanSelectedRowData(cell)
 
             const tableInsert = {
               table: this.table.name,
               schema: this.table.schema,
-              data: fixed,
+              data: this.parseSqlCopyData(fixed),
             }
             const query = await this.connection.getInsertQuery(tableInsert)
```

Comment:
Probably this logic should be moved to `getInsertQuery` so it can be database-specific. Maybe an additional parameter like `stripAutoIncrementColumns` or something